docs/website/docs/guides/ml-frameworks/pytorch.md

```diff
@@ -322,6 +322,132 @@ their values independently at runtime.
         self.value = new_value
     ```
 
+#### :octicons-code-16: Separating parameters during compilation
+
+In practical scenarios, it is usually unfeasible to embed model parameters
```

Comment:
nit: infeasible is more common. 

nit: instead of "In practical scenarios", maybe put "For large models", since this is the practical scenario where this applies.

Reply:
Ok, will do.